app/src/main/res/values/strings.xml

```diff
@@ -700,7 +700,7 @@
     <string name="charging">Charging</string>
     <string name="clear">Clear</string>
     <string name="close">Close</string>
-    <string name="collasped">Collapsed</string>
+    <string name="collapsed">Collapsed</string>
```

Comment:
This will mess up existing translations.

Reply:
i mess up okay, i didnt plan to push this, i planned to use it on my own 🤦 :